### PR TITLE
Replace Haystack with Resolwe ES API

### PR DESCRIPTION
**Pull request by kostko**
The only missing thing is search in the admin interface which used the `SearchModelAdmin` from Haystack.

Will require `urls.py` update in base.

### CODE REVIEW

**Comment by kostko** [comment]
@mstajdohar can you test if this breaks result ordering requirements?

File: resolwe_bio/kb/elastic_indexes.py [hunk 1]
```diff
@@ -54,3 +81,35 @@ def process_object(self, obj, push=True):
...
+    target_id = dsl.String(index='not_analyzed')
+    relation_type = dsl.String(index='not_analyzed')
+
+    class Meta:
+        """Meta class for mapping search document."""
+
+        index = 'mapping_search'
+
+
+class MappingSearchIndex(BaseIndex):
```
Comment (dblenkus) [inline resolwe_bio/kb/elastic_indexes.py]: Use bulk push:
* override `process_object` with `push=False`
* manually push documents in management command and tests
(see `FeatureSearchIndex` and `insert_features` command for example)

Comment (kostko) [inline resolwe_bio/kb/elastic_indexes.py]: Done.

**Comment by kostko** [comment]
Requires fixes from genialis/resolwe#198.

**Comment by mstajdohar** [comment]
The gene autocomplete works great.

**Comment by kostko** [comment]
Tests currently fail due to genialis/resolwe#199, should be fixed as soon as it is merged.

**Pull request merged**